packages/tools/src/tools/displayTools/utils/pointFunctions.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Fast distance calculation between two points
+ * @param pointA
+ * @param pointB
+ * @returns
+ */
+export function fastPointDistance(pointA, pointB) {
```

Comment:
can we augment the function packages/tools/src/utilities/math/point/distanceToPoint.ts to handle Point3 and use that instead of this?

Reply:
yes